src/vanilla/store.ts

```diff
@@ -198,33 +207,6 @@ const addPendingFunction = (pending: Pending, fn: () => void) => {
   pending[2].add(fn)
 }
 
-const flushPending = (pending: Pending) => {
```

Comment:
needed to move flushPending in to buildStore in order to put invoke recomputeDependents from flushPending.

Reply:
While I'm not sure which is better, I would try keeping it outside. I'm not sure about the future, and we may put "pending" in buildStore. In general, I prefer isolating it.

Reply:
We can keep it outside if we complicate pending by queuing a special function that runs before the others.

But I think I prefer flushPending to be part of the store. To me it makes sense that the commit phase of the transaction is internal to buildStore.